DfE.FindInformationAcademiesTrusts/Pages/Shared/_SearchForm.cshtml

```diff
@@ -4,12 +4,36 @@
   Enter trust name, TRN, group ID, UID, UKPRN or Companies House number
 </div>
 
-<div class="govuk-input__wrapper app-search__form  app-search__form--border">
-  <input class="dfe-search__input govuk-!-width-full" id="@Model.InputId" type="search" autocomplete="off" asp-for="KeyWords" name="keywords">
-  <button class="dfe-search__submit app-search__button app-search__button--blue" type="submit">
-    <svg class="dfe-icon dfe-icon__search app-search__icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false" width="27" height="27">
+@* This is used by the no javascript users *@
+<div class="govuk-input__wrapper app-search--main">
+  <div id="@Model.InputId-no-js-search-container">
+    <input class="app-search__input" id="@Model.InputId" type="search" autocomplete="off" asp-for="KeyWords" name="keywords">
```

Comment:
Do we want to switch autocomplete off in this situation? If we leave it switched on will it show previous search terms? On the [mdn docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete), autocomplete attribute is valid on a search box

I realise this may be from DfE frontend code - but just spotted this!

Reply:
I'm happy to turn it on for the non-js route